source/draw/DrawState.ooc

```diff
@@ -22,11 +20,15 @@ BlendMode: enum {
 }
 
 DrawState: cover {
+	// Used by all
 	target: Image = null
 	inputImage: Image = null
+	// Only used by the write function
+	_originReference := FloatPoint2D new(0.0f, 0.0f) // Start of text
```

Comment:
The text will have the origin as the upper left corner of the first character in reference coordinates.